server/views/layout/default.njk

```diff
@@ -1,27 +1,29 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-  <title>Wellcome Collection</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
-  {% include "_partials/css-head.njk" %}
-  <link rel="stylesheet" href="/assets/css/styleguide.css">
-  {# TODO: - come up with better way of including javscript in the head #}
-  {% include "_partials/js-head.njk" %}
-</head>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <title>Wellcome Collection</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+    {% include "_partials/css-head.njk" %}
+    <link rel="stylesheet" href="/assets/css/styleguide.css" />
```

Comment:
This should be a call to the application.css not the styleguide.css (prob not for this PR though)